migrations/kdvh/dump.go

```diff
@@ -0,0 +1,244 @@
+package kdvh
+
+import (
+	"database/sql"
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	_ "github.com/jackc/pgx/v5/stdlib"
+
+	"migrate/utils"
+)
+
+type DumpConfig struct {
+	BaseDir     string   `short:"p" long:"path" default:"./dumps/kdvh" description:"Location the dumped data will be stored in"`
+	TablesCmd   string   `short:"t" long:"table" default:"" description:"Optional comma separated list of table names. By default all available tables are processed"`
+	StationsCmd string   `short:"s" long:"stnr" default:"" description:"Optional comma separated list of stations IDs. By default all station IDs are processed"`
+	ElementsCmd string   `short:"e" long:"elem" default:"" description:"Optional comma separated list of element codes. By default all element codes are processed"`
+	Overwrite   bool     `long:"overwrite" description:"Overwrite any existing dumped files"`
+	Email       []string `long:"email" description:"Optional email address used to notify if the program crashed"`
+
+	Tables   []string
+	Stations []string
+	Elements []string
+}
+
+func (config *DumpConfig) setup() {
+	if config.TablesCmd != "" {
+		config.Tables = strings.Split(config.TablesCmd, ",")
+	}
+	if config.StationsCmd != "" {
+		config.Stations = strings.Split(config.StationsCmd, ",")
+	}
+	if config.ElementsCmd != "" {
+		config.Elements = strings.Split(config.ElementsCmd, ",")
+	}
+}
```

Comment:
`DumpConfig` contains only user inputs, so maybe I should rename it to `DumpFlags`. 
And yes, `go-flags` has a weird way to parse slices, you need to specify the same flag multiple times. I opened a PR there to allow parsing strings with a delimiter, but I don't know if/when it will be merged. I could split them directly in `Execute`, it's just more variables to pass around

Reply:
Ok, I managed to make it use it my `go-flags` fork instead :smile: